src/topology/generic_pipeline.rs

```diff
@@ -109,8 +108,8 @@ where
         let path = env::current_dir()?;
         if !self.processors.is_empty() {
             PROCESSOR_INIT.call_once(|| {
-                pyo3::append_to_inittab!(rotel_python_processor_sdk);
-                pyo3::prepare_freethreaded_python();
+                // pyo3::append_to_inittab!(rotel_python_processor_sdk);
+                // pyo3::prepare_freethreaded_python();
```

Comment:
Was this supposed to stay commented out?

Reply:
Good 👁️, moved the init logic